jvm-packages/xgboost4j/pom.xml

```diff
@@ -47,7 +47,7 @@
           <groupId>org.scalatest</groupId>
           <artifactId>scalatest_${scala.binary.version}</artifactId>
           <version>3.0.5</version>
-          <scope>compile</scope>
+          <scope>provided</scope>
```

Comment:
this should be test ?

Reply:
Ideally, yes. However, the `GpuTestSuite` class references scalatest annotations from the `main` source set.

Reply:
hmmm? currently the build failed with 

```
[2021-02-24T10:51:04.637Z] [ERROR] /workspace/jvm-packages/xgboost4j-gpu/src/main/java/ml/dmlc/xgboost4j/java/GpuTestSuite.java:[23,21] package org.scalatest does not exist

[2021-02-24T10:51:04.637Z] [ERROR] /workspace/jvm-packages/xgboost4j-gpu/src/main/java/ml/dmlc/xgboost4j/java/GpuTestSuite.java:[25,2] cannot find symbol
```

Reply:
which build? the github CI checks are failing on an unrelated problem in the jenkins stage...

Reply:
Ping...

Reply:
⏳